test/Dialect/FIRRTL/emit-omir.mlir

```diff
@@ -330,6 +330,77 @@ firrtl.circuit "SRAMPaths" attributes {annotations = [{
 // CHECK-SAME:    #hw.innerNameRef<@Submodule::[[SYMMEM2:@[a-zA-Z0-9_]+]]>
 // CHECK-SAME:  ]
 
+//===----------------------------------------------------------------------===//
+// Make SRAM Paths Absolute with existing NLA (`SetOMIRSRAMPaths`)
+//===----------------------------------------------------------------------===//
```

Comment:
This test is copied from above (almost verbatim including all the comments), is it necessary or is it sufficient to just include the modifications to the above test to add symbols? Can you use a new, narrower test if not?